openslides/core/config_variables.py

```diff
@@ -130,6 +130,19 @@ def get_config_variables():
         group='General',
         subgroup='Export')
 
+    yield ConfigVariable(
+        name='general_export_pdf_fontsize',
+        default_value='10',
+        input_type='choice',
+        label='Standard font size in PDF',
+        choices=(
+            {'value': '10', 'display_name': '10'},
```

Comment:
Why don't you want to enable more font sizes by enabling a big range instead of this small choice. Minimum 6. Maximum 64 or even no maximum.